index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Addisalem seifu Portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e1cc58c746.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    
+    <section class="front_page">
+        <div class="nav__wrapper">
+            <nav class="nav_bar">
+                <h2 class="logo">My Logo</h2>
+                <ul class="menue_bar">
+                    <li class="bar1"></li>
+                    <li class="bar1"></li>
+                    <li class="bar1"></li>
+                </ul>
+                <ul class="menue_bar_desktop">
+                    <li class="nav_list"><a href="#" class="nav_link">Portofolio</a></li>
+                    <li class="nav_list"><a href="#" class="nav_link">About</a></li>
+                    <li class="nav_list"><a href="#" class="nav_link">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="main__wrapper">
+            <div class="porto_wrapper">
+                <h1 class="headline">I'm Addisalem Glad to see you!</h1>
+                <p class="description">I am a software developer! I can help you build a product, feature or website look through some of my work and experiance! if you like what you see or have a project you need coded don't hesitate to contact me. </p>
+                <h2 class="connect_wrapper"><a class="contact" href="#">LETS CONNECT</a></h2>
+                <ul class="contact_us_wrapper1">
+                    <li><a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a></li>
+                    <li><a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-medium-m"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-github-square"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-angellist"></i></a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    
+    <section class="grid__wrapper">
+        <div class="covers">
+            <img class="img__class" id="pic__2" src="./img/Snap.png" alt="">
+            <img class="img__class" id="pic__1" src="./img/pic1.png" alt="">
```

Comment:
The image for the first project is broken. KIndly see the screenshot below as a reference
![image](https://user-images.githubusercontent.com/35504149/134526833-d52f7b24-7bc1-47f4-8430-c510b419eaf3.png)
.